### PR TITLE
Improve error for PRs from forks

### PR DESCRIPTION
**Pull request by drigz**
This has confused us a few times.

### CODE REVIEW

**Comment by drigz** [comment]
Nicer error here: https://github.com/googlecloudrobotics/core/actions/runs/5240897168/jobs/9462295225?pr=160

**Review by ensonic — APPROVED** [review]
Thanks!

File: .github/workflows/check-bazel.yml [hunk 1]
```diff
@@ -22,11 +22,19 @@ jobs:
         env:
           ROBCO_INTEGRATION_TEST_GITHUB_ROBOT_JSON_KEY: ${{ secrets.ROBCO_INTEGRATION_TEST_GITHUB_ROBOT_JSON_KEY }}
         run: |
+          if [[ -z "$ROBCO_INTEGRATION_TEST_GITHUB_ROBOT_JSON_KEY" ]] ; then
+            echo >&2 "This PR appears to be from a fork, rather than from the primary repo."
```
Comment (faximan) [inline .github/workflows/check-bazel.yml:26]: nit: "fork or authored by a non-org member"

**Review by faximan — COMMENTED** [review]
Thanks Rodrigo! Could you also remove https://github.com/googlecloudrobotics/core/blob/main/.github/ci/common.sh#LL14C8-L14C73 as it seems broken/insufficient?

I am afraid this means we have no solution for supporting presubmits from external contributors (non-org members). I can file a tracking bug about this.

**Comment by drigz** [comment]
@faximan PTAL, looks like I forgot this PR.

**Review by faximan — APPROVED** [review]

**Pull request merged**